_data/sites.json

```diff
@@ -5787,6 +5787,24 @@
         ]
     },
 
+    {
+        "name": "GameStop",
+        "url": "https://www.bollyinside.com/articles/how-to-delete-a-gamestop-account/",
```

Comment:
We ask for a page that is linked to the service itself. This is a tutorial on a website not tied to Gamestop, so either provide a FAQ on the gamestop itself, or point to the best starting point on the gamestop website and describe the deletion process in the notes.

Reply:
The reason why I choose that URL is because GameStop doesn't seem to provide their own guide and it the best one I found. I can close this PR if you insist on including an official guide.

Reply:
I think it's better to add a guide from third-party source than no source at all.

Reply:
I'd be fine just putting GameStop homepage and then describing on the notes steps like the tutorial

Reply:
> You would have to write an E-Mail to GameStop in order to delete you account. Should I therefore add a mailto: link?

There is a separate (optional) field for email, just use that instead of a mailto link, @WebSnke . [See CONTRIBUTING.md](https://github.com/jdm-contrib/jdm/blob/master/CONTRIBUTING.md)